common/txmgr/types/fee_estimator.go

```diff
@@ -15,22 +16,21 @@ const (
 )
 
 // PriorAttempt provides a generic interface for reading tx data to be used in the fee esimators
-type PriorAttempt[FEE any, HASH any] interface {
+type PriorAttempt[FEE any, TX_HASH types.Hashable] interface {
```

Comment:
We'll also use BlockHash in some places.
Whats the best way you want to name these:
TxHash: TH or TX_HASH
BlockHash: BH or BLOCK_HASH

?

Reply:
i'm good with that. I'm mostly just saying that `PriorAttempt` only needs knowledge of one hash so there's no need to differentiate. for types that use both, then `TH` and `BH` are fine to me

Reply:
But on the other side, if we are using TH and BH at most places, its more consistent to use those names at all places in txmgr

Reply:
After making many changes, I feel that single letter acronyms are just confusing in TXM. Because we have things like Tx, TxAttempt, TxHash, BlockHash, Address, Receipt, Fee etc etc.
So it is better to use descriptive names in my opinion.